hibernate-orm-multi-tenancy-schema-quickstart/src/test/java/org/acme/hibernate/orm/IdeTestApp.java

```diff
@@ -0,0 +1,26 @@
+package org.acme.hibernate.orm;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+/**
+ * Workaround to allow launching the Quarkus application inside the IDE.
+ * 
+ * @see https://github.com/quarkusio/quarkus/issues/2143
+ */
+@Disabled("You can enable this pseudo test to run the app in your IDE")
+@QuarkusTest
+public class IdeTestApp {
```

Comment:
This issue appears to have been fixed. I'd suggest removing this class from both Maven modules?